src/RiakClient/Extensions/StringExtensions.cs

```diff
@@ -24,7 +24,10 @@ namespace RiakClient.Extensions
     using System.Web;
     using Util;
 
-    internal static class StringExtensions
+    /// <summary>
+    /// Extension methods to assist with conversion between Riak byte[] and strings.
+    /// </summary>
+    public static class StringExtensions
```

Comment:
I think we have users that use the `ToRiakString/FromRiakString` methods, as it's the easy way to serialize.